src/main/java/backend/resource/TurboMilestone.java

```diff
@@ -83,6 +86,59 @@ public TurboMilestone(TurboMilestone milestone) {
     private void ______METHODS______() {
     }
 
+    public boolean isOverdue() {
+        return dueDate.isPresent() && dueDate.get().isBefore(LocalDate.now());
+    }
+
+    public boolean hasOpenIssues() {
+        return openIssues > 0;
+    }
+
+    /**
+     * A milestone is ongoing if it is not due yet or it is overdue but still has open issues.
```

Comment:
Update the comment here to reflect that we include the condition that the milestone must be open to be ongoing.